cobalt/black_box_tests/testdata/pointer_event_on_cropped_element_test.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <title>Cobalt pointer event test on cropped element</title>
+    <script src='black_box_js_test_utils.js'></script>
+    <script src='pointer_event_test_utils.js'></script>
+    <style>
```

Comment:
Where is the cropping happening?

Reply:
The large bottom part of "upper_element" is being hidden by the "lower_element".

Reply:
Talked offline, we'll want another test for cropping via `overflow: scroll`, but this is also fine to add